docs/src/tutorials/conic/simple_examples.jl

```diff
@@ -245,9 +285,13 @@ example_minimum_distortion()
 
 # ## Robust uncertainty sets
 
-# This example computes the Value at Risk for a data-driven uncertainty set;
-# see "Data-Driven Robust Optimization" (Bertsimas 2013), Section 6.1 for
-# details. Closed-form expressions for the optimal value are available.
+# This example computes the Value at Risk for a data-driven uncertainty set.
+# Closed-form expressions for the optimal value are available.
+# For more details, see:
+
+# Bertsimas, D., Gupta, V., & Kallus, N. (2018). 
```

Comment:
Vale failed because `Kallus` isn't a recognized word in our dictionary. The solution is to add it here:
https://github.com/jump-dev/JuMP.jl/blob/203a5c77383dc8336d7b3fd24048fc8e9b04d9ff/docs/styles/Vocab/JuMP-Vocab/accept.txt#L148-L149